src/components/ListingCard/styles.tsx

```diff
@@ -1,32 +1,44 @@
 import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import { H4 } from '@/styles/text';
+
+const openSans = OpenSans({ subsets: ['latin'] });
 
 // the card itself
 export const CardBody = styled.div<{ $selected?: boolean }>`
   display: flex;
   flex-direction: column;
-  border: 1px solid lightgray;
+  border: 2px solid ${COLORS.midGrey};
   padding: 1rem;
-  border-radius: 10px;
+  border-radius: 5px;
   transition: 150ms;
   cursor: pointer;
   gap: 1rem;
 
-  ${({ $selected }) => $selected && `border-color: #097A62`};
+  ${({ $selected }) => $selected && `border-color: ${COLORS.primaryBlue}`};
 
   &:hover {
-    box-shadow: 0 4px 4px 1px rgba(0, 0, 0, 0.1);
+    box-shadow: 0 4px 4px 0px ${COLORS.midGrey};
     transform: translateY(-2px);
```

Comment:
This is my fault for not using `rem` before, but do you mind changing the `px` units to `rem` here? 😅